test/compilable/extra-files/json2.out

```diff
@@ -1,3 +1,4 @@
+==== ${RESULTS_DIR}/compilable/json2.out ====
```

Comment:
FYI: there's also the har format (https://code.dlang.org/packages/har).

CC @marler8997

Reply:
Thanks for the link. Seems like other tools use HAR as well (or quite similar formats).

I'll switch to HAR.